addons/l10n_tr/__manifest__.py

```diff
@@ -13,10 +13,10 @@
     * Sihirbaz sizden hesap planı şablonu, planın kurulacağı şirket, banka hesap
       bilgileriniz, ilgili para birimi gibi bilgiler isteyecek.
     """,
-    'author': 'Ahmet Altınışık, Can Tecim',
-    'maintainer':'https://launchpad.net/~openerp-turkey, http://www.cantecim.com',
+    'author': 'Odoo',
     'depends': [
         'account',
+        'l10n_multilang',
```

Comment:
Where is the call to `load_translations` in the post init hook?

Reply:
This boy is almost a year old, it was using the old way of `i18n_extra`, it is fixed now.

Reply:
`l10n_extra` doesn't change anything but ok...

Reply:
If you add a dependency to `l10n_multilang`, you can remove the dependency to `account`

Reply:
I think it is good for information, and it is not changing anything, but done!